tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -65,3 +65,19 @@ Feature: State of the quota
     And user "Alice" has uploaded a file inside space "Project Delta" with content "7 bytes" to "test.txt"
     When user "Alice" uploads a file inside space "Project Delta" with content "00011 bytes" to "test.txt" using the WebDAV API
     Then the HTTP status code should be "507"
+
+
+  Scenario Outline: Check the relative amount of quota of personal space
+    Given user "Admin" has changed the quota of the personal space of "Alice Hansen" space to "10000"
+    And user "Alice" has uploaded file "<file_upload>" to "/demo.txt"
+    When the user "Alice" requests these endpoints with "GET" with basic auth
+      | endpoint    |
+      | <end_point> |
+    Then the HTTP status code should be "200"
```

Comment:
can we also check `ocs` status code if possible?

Reply:
yeah, should be possible with `Then the OCS status code should be "<code>"`